docs/src/transforms.md

```diff
@@ -0,0 +1,39 @@
+# Tranforms
+
+## Functions to use transforms
+
+```@docs
+apply
+revert
+isrevertible
+reapply
+```
```

Comment:
Thinking about it again, I think these should show up in the index.md along with the examples.